app/models/Household.ts

```diff
@@ -9,6 +9,7 @@ import { ForecastParameters } from "./ForecastParameters";
 import { socialRentAdjustmentTypes } from "../data/socialRentAdjustmentsRepo";
 import { Lifetime, LifetimeParams } from "./Lifetime"; 
 
+/** Assumed number of heads per-house */
```

Comment:
I had no idea this is what this referred to 💡 

Maybe it's worth changing to a more meaningful name?